vendor/experimental-render-mode-rehydrate.js

```diff
@@ -0,0 +1,25 @@
+(function() {
+  if (typeof FastBoot === 'undefined') {
+    var current = document.getElementById('fastboot-body-start');
+
+    if (current && Ember.ViewUtils.isSerializationFirstNode(current.nextSibling)) {
```

Comment:
Why not provide a mixin for folks to add in their app.js itself?

Reply:
So long as `Ember.ViewUtils.isSerializationFirstNode` is present consumers of fastboot will never have to worry about this line of code unless fastboot actually encodes the DOM the browser version of Ember receives with the glimmer-vm's serialization element builder.  And if the the `isSerializationFirstNode` triggers true then we know the user has opted in.

Reply:
Is the `Ember.ViewUtils.isSerializationFirstNode` a public API?

Reply:
@kratiahuja It is public.

Reply:
@rondale-sc - Can you tweak this to be safe when `Ember.ViewUtils.isSerializationFirstNode` is not present (e.g. you aren’t using >= some 3.2ish canary build ATM)

Reply:
@rwjblue Updated to ensure `typeof Ember.ViewUtils.isSerializationFirstNode === 'function'`

Reply:
Thank you!